cli/objects/rule.py

```diff
@@ -274,6 +276,68 @@ def _get_jira_additional_labels(
         else:
             return None
 
+    def _get_jira_assignee(self, rule_dict: dict[Any, Any]) -> Optional[str]:
+        """
+        Determines if a Jira Assignee is defined in a rule. If it is, validate it and return the string.
+
+        :param rule_dict: A dictionary object representing a user-defined firewatch rule.
+        :return: A string of the Jira assignee to use in a firewatch rule. If one is not defined, return None
+        """
+        if "jira_assignee" in rule_dict.keys():
+            try:
+                jira_assignee = rule_dict["jira_assignee"]
```

Comment:
Nitpick, as far as I know, I think an exception would only get thrown here is the key didn't exist in the dict, and that's handled by line 286, so the use of try catch here might be unnecessary unless there is a corner case I am not thinking of :)

Reply:
You are correct, my bad! Will update in an upcoming commit.

Reply:
Fixed in [b819732](https://github.com/CSPI-QE/firewatch/pull/23/commits/b819732d735899bd815b3292b116c2abf2d3aa67). Thanks!